samples/enableBucketLifecycleManagement.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * This application demonstrates how to enable Object Lifecycle Management for
+ * a bucket.
+ *
+ * For more information, see the README.md under /storage and the documentation
```

Comment:
I just noticed we say this other places, but what do we mean by under "/storage"? We have one in the root of the repo which would be under "nodejs-storage" if they cloned the repo, and one in samples/. Maybe we should just put the URL to the GitHub-hosted file that we mean?

Reply:
Seems like a relic from the past. Rewrote to just point to lifecycle docs.